frontend/src/app/components/containers/complaints/details/complaint-details-create.tsx

```diff
@@ -89,9 +90,15 @@ export const CreateComplaint: FC = () => {
 
   const [complaintData, applyComplaintData] = useState<ComplaintAlias>();
 
-  const [complaintType, setComplaintType] = useState<string>(
-    agency == "EPO" ? COMPLAINT_TYPES.ERS : COMPLAINT_TYPES.HWCR,
-  );
+  let initialComplaintType: string = COMPLAINT_TYPES.HWCR;
+  if (agency === "EPO") {
+    initialComplaintType = COMPLAINT_TYPES.ERS;
+  } else if (agency === "COS") {
+    initialComplaintType = activeTab === COMPLAINT_TYPES.ERS ? COMPLAINT_TYPES.ERS : COMPLAINT_TYPES.HWCR;
+  }
+
+  const [complaintType, setComplaintType] = useState<string>(initialComplaintType);
+
```

Comment:
@dmitri-korin-bcps you may want to check the users roles rather than using the agency. This would be in line with how you dealt with the filters